Add toggle for using QBI to spatial settings (#781)
spatialos/UnrealGDK: SpatialGDK/Documentation/known-issues.md

| NetMulticast RPCs do not work on Singleton Actors. | 2019-03-13 | [UNR-856](https://improbableio.atlassian.net/browse/UNR-856) | Do not use NetMulticast RPCs on Singleton Actors.|
| Replicated TimelineComponents are not supported. | 2019-03-14 | [UNR-1105](https://improbableio.atlassian.net/browse/UNR-1105) | Use an alternative method of achieving the desired behaviour, such as playing the timeline on a multicast event, or replicating what is changed by the timeline. |
| Generating schema for `UClass`es that are not explicitly loaded/referenced by the currently loaded level is not supported. For example, a hard coded path loaded at run-time will not be detected by the schema generator. | 2019-03-25 |  | Reference the class explicitly from an already loaded class (Level blueprint, GameMode etc). Alternatively, you can open the Blueprints for soft references before generating schema so that they are loaded into memory. |

> | Level streaming is not supported without a significant performance degradation. | 2019-03-26 | [UNR-1163](https://improbableio.atlassian.net/browse/UNR-1163) | There is a "QBI Enabled" option in the SpatialOS GDK for Unreal Runtime Settings. If this is on, level streaming will be supported, but at the cost of poorer performance. |

Reviewer: ```suggestion
| Level streaming is not supported without a significant performance degradation. | 2019-03-26 | [UNR-1163](https://improbableio.atlassian.net/browse/UNR-1163) | Select the **QBI Enabled** checkbox in the SpatialOS GDK for Unreal Runtime Settings to enable or disable level streaming. If the **QBI Enabled** checkbox is ticked, level streaming will be supported, but at the cost of poorer performance. |
```

> | Some SpatialOS related settings made after launching SpatialOS will invalidate the currently launched deployment (e.g. Changing the number of servers, or toggling using Query Based Interest), since they require launching with a particular launch configuration. | 2019-03-26 | [UNR-1190](https://improbableio.atlassian.net/browse/UNR-1190) | Manually stop and start SpatialOS when the config changes.  |

Reviewer: ```suggestion
| Changing some SpatialOS settings after launching a deployment will invalidate the currently launched deployment (e.g. Changing the number of servers, or toggling using Query Based Interest), as these settings modify the launch configuration. | 2019-03-26 | [UNR-1190](https://improbableio.atlassian.net/browse/UNR-1190) | Manually stop and start your deployment after you make any changes that alter the launch configuration.  |
```
Is this text accurate? "since they require launching with a particular launch configuration." doesn't scan well to me.

Author: Yeah, that sounds good. I'd want to clarify this being for a 'local' deployment, but otherwise what you put sounds good
